src/EventStore.Core/ClusterVNodeOptions.cs

```diff
@@ -789,7 +790,7 @@ public record ProjectionOptions {
 			public int ProjectionCompilationTimeout { get; set; } = 500;
 
 			[Description("The time in milliseconds allowed for the executing a handler in a user projection")]
```

Comment:
Perhaps the description of `ProjectionExecutionTimeout` should be updated to mention that this can be overridden per projection?